.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,31 @@
+name: autofix.ci
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - "**.html"
+      - "**.css"
+      - "**.js"
+      - "**.md"
+      - "**.tsx"
+      - "**.json"
+  pull_request:
+    branches:
+      - trunk
+  merge_group:
+    types: [checks_requested]
+  page_build:
+permissions: {}
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+      - run: |
+          yarn
+          yarn prettier . --check
```

Comment:
Only check formatting instead of writing the suggestions. This will eliminate the possibility of conflicts if someone is actively working on a branch and is pushing commits.